contracts/fund/policies/risk-management/AdapterBlacklist.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.6.8;
+
+import "../utils/CallOnIntegrationPreValidatePolicyBase.sol";
+import "../utils/AddressListPolicyMixin.sol";
+
+/// @title AdapterBlacklist Contract
+/// @author Melon Council DAO <security@meloncoucil.io>
+/// @notice A blacklist of adapters to add to a fund's vault
+/// @dev Adapters can be added but not removed from blacklist
```

Comment:
This is no longer true. The blacklist config can't be changed.

Can you please remove this comment from AssetBlacklist also?